inst/config/orderly-web.yml

```diff
@@ -11,7 +11,7 @@ orderly:
   image:
     repo: vimc
     name: orderly.server
-    tag: master
+    tag: vimc-3638
```

Comment:
This needs changing back to `master` once https://github.com/vimc/orderly.server/pull/19 is merged